.dapper

```diff
@@ -116,6 +122,11 @@ pass_credentials "${HOME}/.aws/credentials"
 # Pass through ~/.gcp/osServiceAccount.json so that the container can operate on GCP (if needed).
 pass_credentials "${HOME}/.gcp/osServiceAccount.json"
 
+# Attempt to mount any local replaces specified in `go.mod`.
+while read -a replace; do
+    mount_volume "${replace[3]}" "${DAPPER_SOURCE}/${replace[3]}"
+done < <(sed -n -e '/\breplace\b/p' go.mod)
```

Comment:
This will only work for individual `replace` lines, and will catch non-local replacements.

```suggestion
while read replace; do
    mount_volume "${replace}" "${DAPPER_SOURCE}/${replace}"
done < <(go list -mod=mod -m -f "{{.Replace}}" all | grep -E '^\.\.?/')
```

Reply:
The problem is, we cant guarantee that go is available (and the correct version) outside the container (this is outside the dapper).

As for blocks, we can look for `=>` instead to cover both cases (obviously there will be some handling for different directives).

As for non-local replacements its not a problem since mount_volume ignores ones that dont exist, but for clarity we can test it here as well.

Reply:
Ah yes, indeed, so

```suggestion
while read -a replace; do
    mount_volume "${replace[1]}" "${DAPPER_SOURCE}/${replace[1]}"
done < <(grep -o -E '=>[ ]*[^ ]+' go.mod)
```